app/plugins/response-transformer.md

```diff
@@ -9,6 +9,7 @@ breadcrumbs:
 
 Transform the response sent by the upstream server on the fly on Kong, before returning the response to the client.
 
+Caution: To transform the JSON body, plugin buffers the whole response body in memory instead of streaming it synchronously to the client. Also technically its not possible to the set `content-length` with new body size so it is removed form response headers if any of JSON body transformer configs are set. It may give unwanted results for responses with huge body.
```

Comment:
The proper format for notes is an `alert alert-warning` div with a `Note:` prefix. We use it in a lot of places in the website:

```html
<div class="alert alert-warning">
  <strong>Note on transforming bodies:</strong> Be weary of performing transformations on the responses bodies. In order to parse and modify a JSON body, the plugin needs to retain it in memory, which might cause pressure on the worker's Lua VM when dealing with large bodies (several MBs). Because of Nginx's internals, the `Content-Length` header will not be set when transforming a response body.
</div>
```